src/components/Map.ts

```diff
@@ -46,7 +46,7 @@ export class Map extends Component<MapProps, MapState> {
     // Location of Mendix Netherlands office
     private defaultCenterLocation: LatLng = { lat: 51.9066313, lng: 4.488359 };
     private mapLoader?: GoogleMapLoader;
-    private bounds: google.maps.LatLngBounds;
+    private bounds?: google.maps.LatLngBounds;
```

Comment:
Use a definite assignment assertion instead of undefined
`private bounds!: google.maps.LatLngBounds;`